PocketKit/Sources/Sync/PocketSource.swift

```diff
@@ -524,10 +523,10 @@ extension PocketSource {
             let operation = operations.savedItemMutationOperation(
                 apollo: apollo,
                 events: _events,
-                mutation: TagUpdateMutation(input: TagUpdateInput(id: remoteID, name: name))
+                mutation: TagUpdateMutation(input: TagUpdateInput(id: oldTag.remoteID ?? "", name: name))
```

Comment:
@Gio2018 with the changes in terms of using name over remoteID, does this look okay?